cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -996,18 +996,27 @@ def alternative_titles_doc(self, key, value):
 @model.over("number_of_pages", "^300__")  # item
 def number_of_pages(self, key, value):
     """Translates number_of_pages fields."""
-    val = clean_val("a", value, str)
-    if is_excluded(val):
-        raise IgnoreKey("number_of_pages")
+    val_x = clean_val("x", value, str)
+    val_a = clean_val("a", value, str)
+    if val_x:
+        if val_x == "volume":
+            raise IgnoreKey("number_of_pages")
+        elif val_x == "Phys.Desc." or val_x == "Phys.desc":
+            self["physical_description"] = val_a
+            raise IgnoreKey("number_of_pages")
+    else:
+        if is_excluded(val_a):
+            raise IgnoreKey("number_of_pages")
 
-    parts = extract_parts(val)
-    if parts["has_extra"]:
+        parts = extract_parts(val_a)
+        if parts["has_extra"]:
+            raise UnexpectedValue(subfield="a")
+        if parts["physical_copy_description"]:
```

Comment:
after changes in https://github.com/inveniosoftware/invenio-app-ils/pull/1024
could be simply `self['physical_description']`